pkgs/applications/misc/gramps/default.nix

```diff
@@ -26,10 +44,27 @@ in buildPythonApplication rec {
     owner = "gramps-project";
     repo = "gramps";
     rev = "v${version}";
-    sha256 = "00358nzyw686ypqv45imc5k9frcqnhla0hpx9ynna3iy6iz5006x";
+    sha256 = "sha256-BerkDXdFYfZ3rV5AeMo/uk53IN2U5z4GFs757Ar26v0=";
   };
 
-  pythonPath = with pythonPackages; [ bsddb3 pyicu pygobject3 pycairo ];
+  pythonPath = with pythonPackages; [
+    bsddb3
+    pyicu
+    pygobject3
+    pycairo
+    jsonschema
+    mock
+    lxml
```

Comment:
I guess these packages should go to nativeCheckInputs since they are test dependencies.

Reply:
Are these libraries needed at build time, not at test time?
If so, leave them as they are, but if not, you forgot to move them.